integration/spark-common/src/main/scala/org/apache/spark/sql/catalyst/CarbonDDLSqlParser.scala

```diff
@@ -218,6 +220,45 @@ abstract class CarbonDDLSqlParser extends AbstractCarbonSparkSQLParser {
     )
   }
 
+  def getField(col: StructField): Field = {
+    var columnComment: String = ""
+    var plainComment: String = ""
+    if (col.getComment().isDefined) {
+      columnComment = " comment \"" + col.getComment().get + "\""
+      plainComment = col.getComment().get
+    }
+    val x =
+      if (col.dataType.catalogString == "float") {
```

Comment:
This code is moved from `getFields` method of `CarbonSpark2SqlParser.scala` and made it as reusable method. So that it can be called for newly created column as well. It is done to avoid code duplication.